### PR TITLE
[docs] simple web crawler example

### PR DESCRIPTION
**Pull request by maxpumperla**
We want to continue building more, and more easily accessible examples on our docs. This PR starts that initiative by also setting up the testing harness, so that we can just add more notebooks. No particular reason to start with a crawler, just turned out to be a quick one. We'll work on more relevant examples, like @richardliaw 's AIR example wish list in as a follow-up.

### CODE REVIEW

File: doc/source/ray-overview/examples/web-crawler.ipynb [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "In this example we'll quickly demonstrate how to build a simple web scraper in Python and\n",
+    "parallelize it with Ray Tasks with minimal code changes.\n",
+    "\n",
+    "To run this example locally on your machine, please first install `ray` and `beautifulsoup` with\n",
```
Comment (richardliaw) [inline doc/source/ray-overview/examples/web-crawler.ipynb:18]: let's also specify the version you tested against?

File: doc/source/ray-overview/examples/web-crawler.ipynb [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 19.3 s, sys: 340 ms, total: 19.7 s\n",
+      "Wall time: 25.8 s\n"
+     ]
+    },
+    {
```
Comment (richardliaw) [inline doc/source/ray-overview/examples/web-crawler.ipynb:108]: somehow when this renders, it looks like code, so it'd be nice if we can differentiate somehow...

File: doc/source/ray-overview/examples/web-crawler.ipynb (hunk 1 above)
Comment (maxpumperla) [reply doc/source/ray-overview/examples/web-crawler.ipynb]: done

File: doc/source/ray-overview/examples/web-crawler.ipynb (hunk 2 above)
Comment (maxpumperla) [reply doc/source/ray-overview/examples/web-crawler.ipynb]: yes, good point. I made the output bold for now. We can iterate on that.

**Review by pcmoritz — APPROVED** [review]
Nice, this looks great!

The ecosystem gallery looks a little bit misaligned (e.g. it would probably be better if all the github stars were at the bottom and the logos be a little more aligned) :)

Also for the use case gallery, the logos are sometimes a little bit too close to the top.

The content of the new example looks great!

**Comment by richardliaw** [comment]
tests not passing?

**Comment by maxpumperla** [comment]
@richardliaw should be good now, let's see.

**Review by richardliaw — APPROVED** [review]

**Pull request merged**